commons-packet/commons-packet-service/pom.xml

```diff
@@ -30,14 +30,14 @@
         <kernel.keygenerator.bouncycastle.version>1.3.0-SNAPSHOT</kernel.keygenerator.bouncycastle.version>
         <swagger.version>2.9.2</swagger.version>
 
-        <kernel.core.version>1.3.0-SNAPSHOT</kernel.core.version>
+        <kernel.core.version>1.3.0-beta.1</kernel.core.version>
         <kernel.biometrics.api.version>1.3.0-SNAPSHOT</kernel.biometrics.api.version>
         <kernel.crypto-jce.version>1.3.0-SNAPSHOT</kernel.crypto-jce.version>
```

Comment:
Are we using below:
 <kernel.crypto-jce.version>1.3.0-SNAPSHOT</kernel.crypto-jce.version>

Reply:
not using this property. hence removing it.